packages/integration-sdk-core/src/types/logger.ts

```diff
@@ -44,6 +44,7 @@ export interface PublishInfoEventInput extends PublishEventInput {
 
 export enum IntegrationWarnEventName {
   MissingPermission = 'warn_missing_permission',
+  IngestionLimitEncountered = 'warn_ingestion_limit_encountered',
```

Comment:
Docstrings can help developers understand the purpose of this event at development time. I've suggested a description, but I think you could come up with a better description than mine!

```suggestion
  /**
   * Some J1 integrations are configured to only ingest up to a set number 
   * of entities in a given integration job. For example, the Jira integration 
   * is configured to ingest only 2,000 issues per integration job.
   */
  IngestionLimitEncountered = 'warn_ingestion_limit_encountered',
```

Reply:
sounds good! I will add some. I am probably going to omit the second sentence though. If we change the # issues per project ingested on Jira side, I feel like this is liable to get lost in the weeds and deviate from reality.